qutebrowser/browser/pdfjs.py

```diff
@@ -44,6 +44,63 @@ def __init__(self, path):
         super().__init__(message)
 
 
+def _run_js(browsertab, code):
+    # Execute javascript code globally.
+    if objects.backend == usertypes.Backend.QtWebEngine:
+        from PyQt5.QtWebEngineWidgets import QWebEngineScript
+        browsertab.run_js_async(code, world=QWebEngineScript.MainWorld)
+    else:
+        browsertab.run_js_async(code)
+
+
+def is_pdfjs_page(browsertab):
+    """Check whether the given browsertab displays a pdfjs site.
+
+    Args:
+        browsertab: The qutebrowser.browser.browsertab.AbstractTab instance.
+
+    Return:
+        True if the view displays pdfjs, False otherwise.
+    """
+    if objects.backend == usertypes.Backend.QtWebEngine:
+        url = browsertab.url().url()
+        return url.startswith('qute://pdfjs/web/viewer.html')
+    else:
+        # TODO implement this
```

Comment:
I don't think you need a backend distiction here at all?

Reply:
You're right. I was not sure how webkit handles PDF.js, so I thought some complicated logic is needed.